src/components/UserProfile/UserProfileSettings/index.js

```diff
@@ -71,7 +71,9 @@ class UserProfileSettings extends Component {
             </div>
             <div>
               <h4>Password</h4>
-              <Link>Change Password</Link>
+              <Link>
```

Comment:
Do you need Link here ?

Reply:
Yh chetan, we require that for routing to change password

Reply:
removed the link